code/clue.js

```diff
@@ -1,59 +1,260 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS
+
+// OBJECTS FOR ALL THE SUSPECTS
 
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
   color: 'green',
   description: 'He has a lot of connections',
   age: 45,
-  image: 'assets/green.png',
+  image: "url('assets/green.png')",
```

Comment:
This is good practice.  But stick to ' or "
`'url("assets/green.png")'` would be more consistent